fastchat/train/train_lora.py

```diff
@@ -102,11 +100,16 @@ def train():
         lora_args,
     ) = parser.parse_args_into_dataclasses()
 
-    device_map = None
+    if model_args.flash_attn == True:
+        print("Using Flash Attention!")
```

Comment:
As a stylistic note, we should remove the print statements unless we seek to provide info or warn the user. I think we probably won't need the print statements here and down below since the user is inputting that they seek to use flash attention through the arguments.